lib/spack/spack/test/cmd/install.py

```diff
@@ -906,6 +913,7 @@ def test_cdash_auth_token(tmpdir, mock_fetch, install_mockery, capfd):
             assert "Using CDash auth token from environment" in out
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="does not run on windows")
```

Comment:
Same comment as above. In general, apply these comments to all skips where there is a vague reason string.